jsonvalidation/json_validation.go

```diff
@@ -0,0 +1,73 @@
+package jsonvalidation
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/wrapp/instrumentation/logs"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+func badRequest(w http.ResponseWriter, r *http.Request, message, err string) {
```

Comment:
Thinking if we should have a common error response logic available in this lib, similar to this badRequest thing.